tensorflow_io/video/python/ops/video_ops.py

```diff
@@ -68,10 +68,10 @@ def load_dependency_and_library(p):
     ],
 })
 
-class VideoDataset(data_ops.BaseDataset):
+class VideoDataset(data_ops.Dataset):
   """A Video File Dataset that reads the video file."""
 
-  def __init__(self, filename):
+  def __init__(self, filename, batch=None):
```

Comment:
Add docstring for this and a note on the relation between dataset's own `batch()` since I think users would definitely be confused about this.